src/wallet/wallet.h

```diff
@@ -66,7 +66,7 @@ static const CAmount DEFAULT_FALLBACK_FEE = 100000;
 //! -discardfee default
 static const CAmount DEFAULT_DISCARD_FEE = 10000;
 //! -mintxfee default
-static const CAmount DEFAULT_TRANSACTION_MINFEE = 1000;
+static const CAmount DEFAULT_TRANSACTION_MINFEE = 100000;
```

Comment:
This is 1000 presently on master (https://github.com/vertcoin-project/vertcoin-core/blob/master/src/wallet/wallet.h#L73). Why the change? I remember there were some issues with the fee values in previous releases.

Reply:
Thanks for catching this. I looked at the fixes you did [here](https://github.com/vertcoin-project/vertcoin-core/commit/939e9f2d8d50383b27bef47f21f60a4ffe395252) and [here](https://github.com/vertcoin-project/vertcoin-core/commit/c080428f37eb06bd77c9cc4c1846343034a28597) to ensure they weren't reverted by the merge of upstream changes. But I mixed up two variables. It seems that these parameters are still preserved they way they are in 0.17.1. Reverted my change